deploy/helm/groundlight-edge-endpoint/templates/k3s-memory-config.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "groundlight-edge-endpoint.fullname" . }}-k3s-memory-config
+  namespace: {{ .Values.namespace }}
+  labels:
+    {{- include "groundlight-edge-endpoint.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
```

Comment:
This will replace the config file on every upgrade. Is that what we want? (I guess for existing systems we need that).